README.md

```diff
@@ -128,3 +128,40 @@ Below is an example command
 ```sh
 docker run -v $PWD:$PWD -v /usr/local/bin:/usr/local/bin -w $PWD ghcr.io/gem5/gcn-gpu:v24-0 gem5-vega-se gem5/configs/example/apu_se.py -n 3 -c square
 ```
+
+## Note on running GPU FS
+
+You need not use docker to run simulations in GPU FS mode. You still need docker to compile applications like square. We will be simulating AMD's latest GPU, MI300, in FS mode
```

Comment:
Why do you need a docker here? I've installed rocm in this container. See https://github.com/gem5/gem5/pull/1282/files#diff-4a46b3d1ca68a5308768c3ae88fd7bb425557a9a63cab90840b6c8c73e0473d3R71

Reply:
My bad...I did not see this. Will change it now